frontend/src/components/Login/login.jsx

```diff
@@ -1,76 +1,113 @@
-import React from 'react'
-import './login.css'
-import Logo_orange_tan from '../Assets/logoOrangeTan.svg'
-import Logo_orange from '../Assets/logoOrangeBlack.svg'
-
-const LoginForm = () => {
-    const [email, setEmail] = React.useState('');
-    const [password, setPassword] = React.useState('');
-    const handleSubmit = (e) => {
+import React, { useEffect, useState } from 'react';
+import styles from './login.module.css';
+import './login.module.css';
+import Logo_orange_tan from '../Assets/logoOrangeTan.svg';
+import Logo_orange from '../Assets/logoOrangeBlack.svg';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+
+const Login = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [emailError, setEmailError] = useState('');
+    const [passwordError, setPasswordError] = useState('');
+
+    const handleSubmit = async (e) => {
         e.preventDefault();
-        const loginInfo = {email, password}
-    }
+
+        if (!email) {
+            setEmailError('Please enter an email.')
+        }   else {
+            setEmailError('')
+        }
+
+        if (!password) {
+            setPasswordError('Please enter a password.')
+        }   else {
+            setPasswordError('')
+        }
+
+        if (email && password) {
+            const response = await axios.post('/login', {
+                email: email,
+                password: password
+            })
+        }
```

Comment:
redirect the user if the log in was successful

https://reactrouter.com/en/main/fetch/redirect

```suggestion
        if (email && password) {
            const response = await axios.post('/login', {
                email: email,
                password: password
            })
            
            if (response.statusCode === 200) {
                // redirect
            } else  {
            // errors
            }
            
            
            
        }
```